single_pages/dashboard/store/orders.php

```diff
@@ -199,7 +199,15 @@
 
                     <form action="<?=URL::to("/dashboard/store/orders/updatestatus",$order->getOrderID())?>" method="post">
                         <div class="form-group">
-                            <?= $form->select("orderStatus",$orderStatuses,$order->getStatus());?>
+                          <select id="orderStatus" name="orderStatus" ccm-passed-value="Pending" class="form-control">
```

Comment:
I reckon revert this back to the original $form->select for now. We can make a note in the issues to do the translation on the controller side.